ophyd/signal.py

```diff
@@ -164,11 +164,12 @@ def set_thread():
                                value)
                 success = False
             except Exception as ex:
-                self.log.debug('set_and_wait(%r, %s) failed', self.name, value,
-                               exc_info=ex)
+                self.log.exception('set_and_wait(%r, %s) failed',
```

Comment:
This turns the `debug` message into an `error` message. Was that intentional or was the desire to just remove `exc_info=ex`?

Reply:
Mostly driven by number 2, but I think number 1 is also a correct change as `set_and_wait` failing should be a big deal.